.travis.yml

```diff
@@ -13,6 +13,8 @@ before_install:
   - export DOCKER_CLI_EXPERIMENTAL=enabled
   - sudo service docker restart
   - docker --version
+  - bash install_requirements.sh
+  - bash install-edgetpu.sh
```

Comment:
This will have no effect, since ambianic edge is run inside a docker image and not directly in the CI environment.